packages/tally/src/plugin-details.yml

```diff
@@ -0,0 +1,15 @@
+project:
+  name: Tally
+  iconOption: https://i.ibb.co/T43HWdQ/TALLY-LINK-SHARE-IMAGE.png
+  appLink: https://www.tally.xyz/
+
+tasks:
+  - name: Delegate on Arbitrum
```

Comment:
We can only have one task per `actionPluginId`, so this can be changed to just

```
 - name: Delegate on Tally
   link: https://www.tally.xyz/explore
   iconOption: https://i.ibb.co/T43HWdQ/TALLY-LINK-SHARE-IMAGE.png
   actionPluginId: delegate
```

Reply:
fixed, plz, confirm